src/Uno.UWP/Devices/Sensors/Accelerometer.iOS.cs

```diff
@@ -37,7 +37,11 @@ private static Accelerometer TryCreateInstance()
 
 		private void StartReadingChanged()
 		{
-			_motionManager.StartAccelerometerUpdates(new NSOperationQueue(), AccelerometerDataReceived);
+			var nsoperationqueue = NSOperationQueue.CurrentQueue == null || NSOperationQueue.CurrentQueue == NSOperationQueue.MainQueue
+				? new NSOperationQueue()
+				: NSOperationQueue.CurrentQueue;
+
+			_motionManager.StartAccelerometerUpdates(nsoperationqueue, AccelerometerDataReceived);
```

Comment:
I'm not sure what is the effect of this change, but:

- this logic is duplicated in the following as well? https://github.com/unoplatform/uno/blob/6b686ea22265a11ff0dd359f2725fda81725aea9/src/Uno.UWP/Devices/Sensors/SimpleOrientationSensor.iOS.cs#L21
- should this be in https://github.com/unoplatform/uno/blob/6b686ea22265a11ff0dd359f2725fda81725aea9/src/Uno.UWP/Devices/Sensors/Barometer.iOS.cs#L31 https://github.com/unoplatform/uno/blob/6b686ea22265a11ff0dd359f2725fda81725aea9/src/Uno.UWP/Devices/Sensors/Gyrometer.iOS.cs#L32 https://github.com/unoplatform/uno/blob/6b686ea22265a11ff0dd359f2725fda81725aea9/src/Uno.UWP/Devices/Sensors/Magnetometer.iOS.cs#L36
- We should consider extracting this to a common helper method.

Reply:
@Youssef1313  thanks for the review.

The change looks to prevent creating a new Operation Queue instance (Reusing the current if any is available), and it ensures we don't schedule the Accelerometer operation in the Main Queue.

About the duplicate logic and helper function: I can include these suggestions in another PR I have to refactor some parts of the `Accelerometer`. 

I tend to avoid any refactoring work while fixing a bug, especially when the refactoring touch areas that are not related to where the fix was made.